cfme/tests/containers/test_containers_overview_data_integrity.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from utils import testgen
+from utils.version import current_version
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.node import Node
+from cfme.containers.pod import Pod
+from cfme.containers.service import Service
+from cfme.containers.image_registry import ImageRegistry
+from cfme.containers.image import Image
+from cfme.containers.project import Project
+from cfme.containers.route import Route
+from collections import namedtuple
+from cfme.containers.container import Container
+import time
+from cfme.web_ui import StatusBox
+from utils.providers import list_providers
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda provider: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+DataSet = namedtuple('DataSet', ['object', 'name'])
+
+DATA_SETS = [
+    DataSet(Container, 'Containers'),
+    DataSet(Node, 'Nodes'),
+    DataSet(Project, 'Projects'),
+    DataSet(Pod, 'Pods'),
+    DataSet(Service, 'Services'),
+    DataSet(Image, 'Images'),
+    DataSet(Route, 'Routes'),
+    DataSet(ContainersProvider, 'Providers'),
+    DataSet(ImageRegistry, 'Registries')
+]
+
+
+def get_api_object_counts(providers):
+    out = {
+        ContainersProvider: 0,
+        Container: 0,
+        Node: 0,
+        Pod: 0,
+        Service: 0,
+        ImageRegistry: 0,
+        Image: 0,
+        Project: 0,
+        Route: 0,
+    }
+    for provider in providers:
+        out[ContainersProvider] += 1
+        out[Container] = len(provider.mgmt.list_container())
+        out[Node] += len(provider.mgmt.list_node())
+        out[Pod] += len(provider.mgmt.list_container_group())
+        out[Service] += len(provider.mgmt.list_service())
+        out[ImageRegistry] += len(provider.mgmt.list_image_registry())
+        out[Image] += len(provider.mgmt.list_image())
+        out[Project] += len(provider.mgmt.list_project())
+        out[Route] += len(provider.mgmt.list_route())
+    return out
+
+
+# CMP-9521
+
+
+def test_containers_overview_data_integrity():
+    """Test data integrity of status boxes in containers dashboard.
+    Steps:
+        * Go to Containers / Overview
+        * All cells should contain the correct relevant information
+            # of nodes
+            # of providers
+            # ...
+    """
+    sel.force_navigate('container_dashboard')
```

Comment:
We shouldn't need any more ```force_navigates``` should we? If this destination isn't navmazing'd yet, then it should be